tools/recipes/pahole.recipe

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+download() {
+    git clone -b 'v1.20' --single-branch --depth 1 git://git.kernel.org/pub/scm/devel/pahole/pahole.git pahole_src
+    echo "Pahole download completed"
+}
+
+build() {
+    cd pahole_src
+    mkdir build
+    cd build
+    cmake -D__LIB=lib ..
```

Comment:
We need to build static binaries, so that will involve:
* most likely build all dependencies in the recipe
* ask cmake to create a static binary

You can have a look at the rt-app recipe to see how we build dependencies (one build function called in a separate subshell for each).

Reply:
Resolved in new version. As updated version of pahole is not released yet, at present it uses
    d124926baf2366a2ef9db8ac796874cae2b63831 commit from master branch.